lib/RateLimiterRedis.js

```diff
@@ -98,37 +98,39 @@ class RateLimiterRedis extends RateLimiterStoreAbstract {
 
             return resolve(res);
           });
-      } else if (secDuration > 0) {
-        const incrCallback = (err, result) => {
-          if (err) {
-            return reject(err);
-          }
-
-          return resolve(result);
-        };
-
-        if (typeof this.client.rlflxIncr === 'function') {
-          this.client.rlflxIncr(rlKey, points, secDuration, incrCallback);
-        } else {
-          this.client.eval(incrTtlLuaScript, 1, rlKey, points, secDuration, incrCallback);
-        }
       } else {
-        multi.incrby(rlKey, points)
-          .pttl(rlKey)
-          .exec((err, res) => {
+        if (secDuration > 0) {
+          const incrCallback = (err, result) => {
```

Comment:
Here's another change that was made by an ESLint autofix.  Please let me know if you'd like me to revert it.

Comparing to base:
```diff
- const incrCallback = function(err, result) {
+ const incrCallback = (err, result) => {
```